ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/stores/ledger/ImplicitPartyAdditionIT.scala

```diff
@@ -183,7 +183,7 @@ object ImplicitPartyAdditionIT {
   )
 
   private def textValue(t: String) =
-    VersionedValue(ValueVersions.acceptedVersions.head, ValueText(t))
+    VersionedValue(ValueVersion("5"), ValueText(t))
```

Comment:
Perhaps a comment explaining why we use this version?

Reply:
In fact such piece of code should never use explicit version. 
Should call `ValueVersions.assignVersion`.

fixed